multiqc/multiqc.py

```diff
@@ -322,6 +328,7 @@ def run(
     lint=False,
     make_pdf=False,
     no_megaqc_upload=False,
+    no_versions_section=False,
```

Comment:
When adding CLI flags for config options, the default needs to be merged with the user's config. See below after the comment `# Set up key variables (overwrite config vars from command line)`.